K2Bridge.Tests.UnitTests/KustoDAL/KustoDataAccessTests.cs

```diff
@@ -630,6 +630,77 @@ public async Task GetFieldCaps_WithPercentage_ReturnCorrectQuery()
             calls[0].QueryCommandText.Should().Be(@"testIndexName | sample 10 | summarize buildschema(mydynamic)");
         }
 
+        [Test]
+        public async Task GetFieldCaps_WithMaxHours_ReturnCorrectQuery()
+        {
+            const ulong hours = 4;
+            Func<string, string, Dictionary<string, object>> column = (name, type) =>
+                new Dictionary<string, object> {
+                    { "ColumnName", name },
+                    { "ColumnType", type },
+                };
+
+            var testData = new List<Dictionary<string, object>>() {
+                column("myint", "System.Int32"),
+                column("mydynamic", "System.Object"),
+            };
+            using IDataReader testReader = new DataReaderMock(testData);
+            mockQueryExecutor.Setup(exec => exec.ExecuteControlCommandAsync(It.IsNotNull<string>(), It.IsAny<RequestContext>()))
+                .Returns(Task.FromResult(testReader));
+
+            using IDataReader dynamicResultReader = new DataReaderMock(new List<Dictionary<string, object>>() {
+                new Dictionary<string, object>() {
+                    { "result", JToken.Parse("{\"a\": [\"int\", \"string\"], \"b\": {\"`indexer`\": \"int\"}, \"c\": {\"d\": [{\"e\": \"string\"}, \"int\"]}}") },
```

Comment:
is this  the correct  char? 
` vs '

Reply:
Yes - `indexer` is a special value that indicates an array in the schema.
https://docs.microsoft.com/en-us/azure/data-explorer/kusto/query/buildschema-aggfunction